### PR TITLE
[sample-data] Rename menuItem Author-Login and module Author Menu, fix <p tags

### PR DESCRIPTION
**Pull request by chmst**
Related to #202

### Summary of Changes
The menu item title "Autor Login" seems to be too restrictive. There is lot of contente where users mus be registered, not only as author.
Author Menu has been renamed to Special to Special Menu, as it allows login not only for autors but fur user groups with access "special". 

Fixed <p Tags in Articles. 

grafik

### Testing Instructions
Install Sample Data Plugin on a fresh installation, the menuItem now is "Login". The name "Special Menu" is visible only in backend.

### CODE REVIEW

**Comment by richard67** [comment]
Hmm, "Special Menu" .. not very specific. But I don't have better ideas.

**Comment by richard67** [comment]
The summary of changes in the description of this PR doesn't mention the change of the menu title from "Author Menu" to "Special Menu".

**Comment by richard67** [comment]
@brianteeman Maybe you have an idea?

The menu item title "Author Menu" was misleading because 1. the menu item and items below it are not only for users of group "Authors", and 2. they aren't all related to editing content only. So I agree with this PR here that it's good to change it.

The new title "Special Menu" suggested in this PR here is correct regarding access level "Special", but to me it seems to be a bit unspecific. For a user "Special" can mean anything. Do you have a better idea for a good title for that menu item?

**Review by richard67 — APPROVED** [review]

**Comment by brianteeman** [comment]
In the context of the sample data where you will probably only ever have one user which will be the admin I don't really see a problem with calling it author menu you the only other alternative I can think of to address your concerns would be to call it it admin menu

**Comment by richard67** [comment]
@chmst Do you want to change it back to "Author Menu" or to "Admin Menu"? I'd be ok with both. I can also live with "Special Menu", but less happy, because I think it's maybe too unspecific for the user to get an idea on what it could be good for.

**Review by richard67 — APPROVED** [review]

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 1]
```diff
@@ -28,10 +28,10 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_6_FULLTEXT="Lorem ipsum dolor sit
...
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_6_TITLE="Millions"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_FULLTEXT="<p>Uurnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper artichoke.</p> "
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_INTROTEXT="<p><strong>We love Joomla to the moon and back!</strong><p><p> Thank you to all volunteers who have contributed!</p> "
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_INTROTEXT="<p><strong>We love Joomla to the moon and back!</strong></p><p>Thank you to all volunteers who have contributed!</p> "
```
Comment (richard67) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_INTROTEXT="<p><strong>We love Joomla to the moon and back!</strong></p><p>Thank you to all volunteers who have contributed!</p>"
```
Remove space between closing p tag and the double qoute at the end of the language string.

**Pull request merged**